test/index.test.js

```diff
@@ -10,6 +10,10 @@ describe('glob-parent', function () {
     expect(gp('.*')).toEqual('.');
     expect(gp('/.*')).toEqual('/');
     expect(gp('/.*/')).toEqual('/');
+    expect(gp('//')).toEqual('/');
+    expect(gp('//*')).toEqual('/');
+    expect(gp('.//')).toEqual('./');
+    expect(gp('.//*')).toEqual('./');
```

Comment:
Reviewing this PR again, I think collapsing duplicate slashes is a separate feature (that is breaking) and we need to submit a separate PR